airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -2844,6 +2844,18 @@ def _escape(s: str) -> str:
 
 
 def _bq_cast(string_field: str, bq_type: str) -> None | int | float | bool | str:
+    """
+    Helper method that casts a BigQuery row to the appropriate data types.
+    This is useful because BigQuery returns all fields as strings.
+    """
+    warnings.warn(
+        "This function is deprecated and will be Remove in future. Please use `bq_cast` function",
+        DeprecationWarning,
+    )
+    return bq_cast(string_field=string_field, bq_type=bq_type)
```

Comment:
I’d not do this at all, the function is private and not covered by backward compatibility policies.

Reply:
hmm, just wanted to avoid breaking any external lib depending on it. I have removed this warning 5c7ef1579af5d3d4a05d83fa43488dd4de9ac8f6